src/Microsoft.HttpRepl/Commands/BaseHttpCommand.cs

```diff
@@ -557,21 +581,38 @@ protected override string GetHelpDetails(IShellState shellState, HttpState progr
 
         public override string GetHelpSummary(IShellState shellState, HttpState programState)
         {
+#pragma warning disable CA1308 // Normalize strings to uppercase
             return $"{Verb.ToLowerInvariant()} - Issues a {Verb.ToUpperInvariant()} request";
+#pragma warning restore CA1308 // Normalize strings to uppercase
         }
 
         protected override IEnumerable<string> GetArgumentSuggestionsForText(IShellState shellState, HttpState programState, ICoreParseResult parseResult, DefaultCommandInput<ICoreParseResult> commandInput, string normalCompletionString)
         {
+            if (programState is null)
+            {
+                throw new ArgumentNullException(nameof(programState));
+            }
+
             List<string> results = new List<string>();
 
-            if (programState.Structure != null && programState.BaseAddress != null)
+            if (!(programState.Structure is null) && !(programState.BaseAddress is null))
```

Comment:
Where did we land on the `foo is object` vs `foo != null` conversation?

Reply:
Not sure where we landed. But I went ahead and changed it for this repo. I was tempted to use the newer `if (foo is { })` but that required upgrading to C# 8 and I didn't want to make that change in this PR :)